lib/client.js

```diff
@@ -1,9 +1,11 @@
 /* global URL */
-import version from 'version'
+import pkgJson from '../package.json'
```

Comment:
This builds to the same code with optimisations enabled but doesn't require webpack as an intermediary, meaning that the entire module can be bundled into another webpack bundle with source maps intact.